### PR TITLE
Added image and imageview implementation for winforms

### PR DESCRIPTION
**Pull request by obulat**
Signed-off-by: obulat <obulat@gmail.com>

I added image and imageview implementation for winforms, testing it using the imageview app in the examples. The images show, but alignment and scaling don't work yet. There is also a border on imageview for testing purposes.

## PR Checklist:

- [x] All new features have been tested
- [ ] All new features have been documented
- [x] I have read the **CONTRIBUTING.md** file
- [x] I will abide by the code of conduct

### CODE REVIEW

File: src/winforms/toga_winforms/widgets/imageview.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+    def set_image(self, image):
+        if image:
+            # Workaround for loading image from url
+            if isinstance(image._impl.native, str):
+                self.native.Load(image._impl.native)
+            else:
+                self.native.Image = image._impl.native
+        else:
+            # When do we have no image here???
```
Comment (freakboy3742) [inline src/winforms/toga_winforms/widgets/imageview.py]: Consider the case of an image browser - there's a list of image files that can be displayed, and an image view. By default, no image is selected, so no image is displayed.

File: src/winforms/toga_winforms/widgets/imageview.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        self.native.BorderStyle = WinForms.BorderStyle.Fixed3D
+
+    def get_image(self):
+        return self.native.image
+
+    def set_image(self, image):
+        if image:
+            # Workaround for loading image from url
+            if isinstance(image._impl.native, str):
+                self.native.Load(image._impl.native)
```
Comment (freakboy3742) [inline src/winforms/toga_winforms/widgets/imageview.py]: Out of interest - is this a blocking load? If the URL takes a while to resolve and download, does the rest of the GUI freeze?

File: src/winforms/toga_winforms/widgets/image.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    def __init__(self, interface):
+        self.interface = interface
+        self.interface.impl = self
+
+    def load_image(self, path):
+        print('[Image.py] Loading image, path: ', path)
+        if path.startswith('http://') or path.startswith('https://'):
+            print('[Image.py] Temporarily image is the path')
+            self.native = path
+            print('[Image.py] native is: ', self.native)
```
Comment (freakboy3742) [inline src/winforms/toga_winforms/widgets/image.py]: These debug statements obviously need to be cleaned up :-)

File: src/winforms/toga_winforms/widgets/imageview.py (hunk 2 above)
Comment (obulat) [reply src/winforms/toga_winforms/widgets/imageview.py]: When a 40mb image url is added to the example app, it takes a while to open the app. So, I guess, it's blocking? I couldn't implement on_selection...

**Comment by obulat** [comment]
Handling of alignment should be done in another PR :)

**Pull request merged**